lib/ansible/modules/system/firewalld.py

```diff
@@ -73,11 +73,13 @@
         operate on firewalld configs when it's not running (requires firewalld >= 3.0.9)
     required: false
     default: null
+    choices: ["yes","no"]
```

Comment:
instead of choices, just set the `type: boolean` as there are more than 'yes'/'no', it can also be true/false, etc

Reply:
Fair point. :+1: